tools/scripts/ansible/hostname-playbook.yml

```diff
@@ -0,0 +1,6 @@
+- hosts: prod
+  become: yes
+  tasks:
+    - name: Set the hostname
+      hostname:
+         name: st.unicode.org
```

Comment:
should this be `{{ inventory_hostname }}` instead?

Reply:
Yes! I hunted around for a way to get the "current" host, but didn't find this. I'll make a follow-up PR with that change, thanks!

I don't know if there's any reason to do the same for other hosts (like smoketest), so I limited it to "prod"